docs/testbed/README.testbed.ocs_data_plan_deployment.md

```diff
@@ -0,0 +1,262 @@
+1. On **host**, use netplan to build up bridge on network card.
+    ```
+    cat /etc/netplan/01-network-br.yaml 
+    network:
+    version: 2
+    renderer: NetworkManager
+    ethernets:
+        enp0s3:
+        dhcp4: no 
+    bridges:
+        br0:
+        addresses: 
+            - 192.168.1.78/24
+            - '2001:db8:1::1/64'
+        gateway4: 192.168.1.1
+        interfaces: [enp0s3]  
+        parameters:
+            stp: true         
+            forward-delay: 3  
+    ```
+    - Apply netplan.
+    ```
+    sudo netplan apply
+    ```
+    - Check that the bridged network created on the host is successful.
+    ```
+    # brctl show
+    bridge name	bridge id		STP enabled	interfaces
+    br0		8000.00e04c68007f	yes		enp0s3
+    docker0		8000.86cd26814e67	no		veth9f77700
+    ```
+2. Build up network for PTF container.
+    Create a virtual veth pair to help the PTF achieve network communication.
+    First, configure on ***host***
+    - Create veth pair on host
+    ```
+    sudo ip link add veth0 type veth peer name veth1
+    ```
+    - Add one of the veth interfaces, veth1, to the created bridged network br0.
+    ```
+    sudo brctl addif br0 veth1
+    ```
+    - Check the bridge configuration of br0 for veth1
+    ```
+    # brctl show
+    bridge name	bridge id		STP enabled	interfaces
+    br0		8000.00e04c68007f	yes		enp0s3
+                                                            veth1
+    docker0		8000.86cd26814e67	no		veth9f77700
+    ``` 
+    - Set veth1 up
+    ```
+    sudo ip link set dev veth1 up
+    ``` 
+    - Search the process ID of the PTF Docker.
+    ```
+    docker inspect --format '{{ .State.Pid }}' ptf_1
+    187422
+    ```
+    - Configure veth0 to PTF by netns method for network namespace configuration.
+    ```
+    sudo ip link set veth0 netns 187422
+    ```
+    Then, need to login in **PTF Docker** for network configuration
+    - Login into PTF to configure router
+    ```
+    docker exec -it ptf_1 bash
+    ip addr add 192.168.1.101/24 dev veth0
+    ip addr add 2001:db8:1::100/64 dev veth0
+    ip link set dev veth0 up
+    ```
+    - Set Router in PTF
+    ```
+    ip route add default via 192.168.1.78 dev veth0
+    ```
+## Configure Inventory for OCS
+In **sonic docker**, we should create OCS inventory.
+Create a new OCS Inventory for OCS Testbed under */data/sonic-mgmt/ansible*
+
+```
+# vi ocs
+# cat ocs
+all:
+  children:
+    ocs:
+      children:
+        sonic:
+          children:
+            sonic_ocs:
+        servers:
+          children:
+            vm_host:
+    ptf:
+      vars:
+        ansible_ssh_user: root
+        ansible_ssh_pass: 123456
+      hosts:
+        ptf_1:
+          ansible_host: 192.168.1.101
+sonic_ocs:
+  vars:
+    hwsku: SKU checked on OCS
+  hosts:
+    ocs-sonic:
+      ansible_host: 192.168.1.3
+      ansible_ssh_user: username
+      ansible_ssh_pass: password
+vm_host:
+  hosts:
+    virtualpc:
+      ansible_host: 192.168.1.2
```

Comment:
Update the example `ocs` and `testbed-ocs.yml` to match the content here in doc?